samples/bootloader/README.rst

```diff
@@ -154,25 +172,49 @@ Complete the following steps to add the bootloader sample as child image to your
    The build process creates two images, one for the bootloader and one for the application, and merges them together.
 #.  Select :guilabel:`Build` -> :guilabel:`Build and Run` to program the resulting image to your device.
 
+Building on the command line
+============================
 
-Testing
-=======
+You can easily add an immutable bootloader to most Zephyr or |NCS| samples by enabling :option:`CONFIG_SECURE_BOOT` in the sample's ``prj.conf`` file, or directly to the build command as follows:
 
-To test the bootloader sample, add it to any other sample, build and program that sample as described in :ref:`bootloader_build_and_run`.
+.. parsed-literal::
 
-Then test it by performing the following steps:
+   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -DCONFIG_SECURE_BOOT=y
 
-1. |connect_terminal|
-#. Reset the board.
-#. Observe that the kit prints the following information::
+For more information, see :ref:`gs_programming_cmd`.
+
+Bootloader Overlays
+-------------------
+
+Overlays specific to bootloaders can be used to further modify the bootloader child image when compiling with an application:
```

Comment:
Kconfig fragments, not overlays.

Reply:
Addressed in [this commit](https://github.com/nrfconnect/sdk-nrf/pull/3607/commits/8f78f39ea2d00a43e1149293706416747f825acd) for #3607.

Also updated wording from overlay to "fragment" or "Kconfig fragment" in other documents being updated in #3607.